src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -1114,6 +1107,23 @@ private bool ShouldExecuteCodeBlockActions(AnalysisScope analysisScope, ISymbol
             return false;
         }
 
+        private bool ShouldExecuteOperationBlockActions(AnalysisScope analysisScope, ISymbol symbol)
```

Comment:
Minor: You can probably make `ShouldExecuteCodeBlockActions` generic `ShouldExecuteCodeBlockActions<TBlockStartActionsByAnalyzer, TBlockActionsByAnalyzer>` and avoid duplicated code.